lib-api/src/main/java/org/ergoplatform/appkit/BoxOperations.java

```diff
@@ -205,24 +205,37 @@ public SignedTransaction putToContractTx(
     public UnsignedTransaction putToContractTxUnsigned(
         BlockchainContext ctx,
         ErgoContract contract) {
+
+        return buildTxWithTransactionBuilder(ctx, txB -> {
```

Comment:
BTW, those "DefaultInputs" can actually be configured (or optionally configurable with good defaults), since BoxOperations class is already looks and feels like a builder. This will make the methods more reusable.

Reply:
Thought about it, but I am not sure if this makes any sense - mainly because I never needed another input config. That doesn't mean there's no use case, but I would prefer to wait for such a use case so that the interface design will really match actual needs.

Reply:
One problem with this approach is that changes in interfaces will break dApps.

Reply:
The current approach? No, there is no change in the interface. We've only added a new method. I've changed the test case just to use the new method.

I have thought some more about changing something with the inputs. We should wait until there really is an actual use case. At the moment, all inputs can already be changed by setting a loader (changes input boxes) or fee which should fit for 99% of use cases.